chef-config/lib/chef-config/config.rb

```diff
@@ -113,6 +113,8 @@ def self.is_valid_url? uri
             File.expand_path("..", path)
           end
         end
+      elsif configuration[:cookbook_artifact_path]
```

Comment:
I don't know what that is used elsewhere here, it's not a local variable in this scope so you don't need an explicit receiver. Works fine in my testing.